agora/Agora/Proposal.hs

```diff
@@ -55,7 +55,7 @@ import Generics.SOP qualified as SOP
 import Plutarch.Api.V1 (PCredential, PMap, PValidatorHash)
 import Plutarch.Api.V1.AssocMap qualified as PAssocMap
 import Plutarch.Api.V2 (
-  KeyGuarantees (Unsorted),
+  KeyGuarantees (..),
```

Comment:
List it out please! It's part of the standards now. Let's at least enforce it for the main `agora` code.

Reply:
Ahh, that's a little bit annoying.